src/main/java/org/prebid/server/execution/LogModifier.java

```diff
@@ -0,0 +1,31 @@
+package org.prebid.server.execution;
+
+import io.vertx.core.logging.Logger;
+
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.BiConsumer;
+
+public class LogModifier {
+
+    private AtomicInteger errorLevelCount;
+    private volatile BiConsumer<Logger, String> defaultLogModifier;
+    private volatile BiConsumer<Logger, String> logModifier;
+
+    public LogModifier(BiConsumer<Logger, String> defaultLogModifier) {
+        this.defaultLogModifier = defaultLogModifier;
+    }
+
+    public void setLogModifier(int requestCount, BiConsumer<Logger, String> logModifier) {
```

Comment:
Since this class has only 2 methods, mb it make sense to rename those methods to `set(...)` and `get()` ?
Minor. Pls flip method arguments. We should follow the rule - what we set here is more valuable.